evm.md

```diff
@@ -182,7 +185,78 @@ Our semantics is modal, with the initial mode being set on the command line via
 
 State Stacks
 ------------
+### The BlockchainStack
+
+The `blockList` cell stores a list of previous block and network states.
+-   `#pushBlockchainState` saves a copy of the block state and network state on the `blockList`.
+-   `#getBlockchainState(Int)` restores a blockchain state for a given block number.
+
+```k
+    syntax BlockchainItem ::= "{" NetworkCellFragment "|" BlockCell "}"
+ // -------------------------------------------------------------------
```

Comment:
Looks like you're using a mix of `CellFragment` and `Cell`. I would personally just use `BlockCellFragment`, as I know from other projects that it works and it provides a simpler user-experience in my opinion.

Reply:
I used `BlockCell` to be able to identify a block in `<blockList>` using the `BLOCKNUM` in `#getBlockchainState(Int)`

Reply:
You can still do that by matching on `<block>-fragment ... </block>-fragment`

Reply:
Actually, I'm confused about how `#getBlockchainState` is working. (1) is it correct to index based on `BLOCKNUM`, and (2) should we be dropping blocks as we go, or just going through and finding the one we need. @dwightguth seemed to think we should be keeping all the blocks. Would it make more sense to use configuration:

```
<blocks>
  <block multiplicity="*" type="Map">
    <number> ... </number>            // Index on block number if needed
    ...
    <txList> ... </txList>                   // Store list of relevant transaction identifiers here, but leave transactions in `<network>` state?
  </block>
</blocks>
```